DEVELOPMENT.md

```diff
@@ -117,21 +117,26 @@ Your user must be a cluster admin to perform the setup needed for Knative.
 
 The value you use depends on
 [your cluster setup](https://www.knative.dev/docs/install/): when using
-Minikube, the user is your local user; when using GKE, the user is your GCP
-user.
+Minikube or the Kubernetes under local Docker desktop, the user is your local user;
+when using GKE, the user is your GCP user.
 
 ```shell
 # For GCP
 kubectl create clusterrolebinding cluster-admin-binding \
   --clusterrole=cluster-admin \
   --user=$(gcloud config get-value core/account)
 
-# For minikube
+# For minikube or Kubernetes under docker-desktop
 kubectl create clusterrolebinding cluster-admin-binding \
   --clusterrole=cluster-admin \
   --user=$USER
 ```
 
+### Resource allocation for Kubernetes
+
+Please allocate sufficient resources for Kubernetes, no matter it is in GKE, Minikube, docker-desktop, etc.
+We recommend to allocate at least 8 CPUs and 8G memory.
```

Comment:
I see that Istio recommends 8GB memory for Minikube ( https://istio.io/docs/setup/kubernetes/prepare/platform-setup/minikube/ ), but they recommend 4 CPUs. Did 8 CPUs come from trial-and-error? I would be interested to know if we could recommend lower here.

If you have any commands or docs you can link that shows how to allocate this on docker-desktop or minikube I am sure others would find that helpful.

Reply:
I run on 6 CPUs on Docker for Mac. Haven‘t tried to push lower yet.

Reply:
@dgerd 4 CPUs worked fine with Istio, but it does not fit the controller, networking-certmanager, and networking-istio. I ended up with them stuck in pending, if 4 CPUs were allocated.
This is the requirement for ultimate Knative serving, not just Istio.
@markusthoemmes I tuned down to 6 CPUs as well. So far so good.  
So, 6 CPUs and 8G memory??

Reply:
@dgerd 4 CPUs worked fine with Istio, but it does not fit the controller, networking-certmanager, and networking-istio. I ended up with them stuck in pending, if 4 CPUs were allocated.
This is the requirement for ultimate Knative serving, not just Istio.
@markusthoemmes I tuned down to 6 CPUs as well. So far so good.
So, 6 CPUs and 8G memory sound good??